taichi/rhi/opengl/opengl_device.cpp

```diff
@@ -549,6 +549,15 @@ void GLDevice::dealloc_memory(DeviceAllocation handle) {
   check_opengl_error("glDeleteBuffers");
 }
 
+GLint GLDevice::get_devalloc_size(DeviceAllocation handle) {
+  glBindBuffer(GL_ARRAY_BUFFER, handle.alloc_id);
+  check_opengl_error("glBindBuffer");
+  GLint size = 0;
+  glGetBufferParameteriv(GL_ARRAY_BUFFER, GL_BUFFER_SIZE, &size);
+  check_opengl_error("glGetBufferParameteriv");
+  return size;
```

Comment:
nit: conventionally we call the binding function again with ID=0 to release the binding point.
```suggestion
  return size;
  glBindBuffer(GL_ARRAY_BUFFER, 0);
```